src/components/PanelSidetray/PanelSidetray.vue

```diff
@@ -0,0 +1,129 @@
+<template lang="html">
+    <div class="panel-sidetray">
+        <svg-icon-reel class="svg" />
+        <nuxt-link
+            v-for="(item, i) in items"
+            v-if="item.featuredImage.sourceUrl"
+            :key="item.id"
+            :to="item.uri"
+        >
+            <wp-image
+                :image="item.featuredImage"
+                :aspect-ratio="56.25"
+                :class="{ active: i == activeIndex }"
+            >
+                <div class="scrim">
+                    <h4 v-if="i == activeIndex" class="title">
+                        Currently Viewing
+                    </h4>
+                    <h4 v-else class="title" v-html="item.title" />
+                </div>
+            </wp-image>
+        </nuxt-link>
+    </div>
```

Comment:
How you do this, and have it work correctly is this:

```
<div class="panel-sidetray"> // height --unit-100vh, position: fixed, right: 0; transform: translateX(100%);
      <svg-film> // Position: absolute, -100px;
       
      <div class="items"> // Height 100%, overflow-y: auto;
              <nuxt-link/>
              <nuxt-link/>
              <nuxt-link/>
      </div>
</div>
```

Getting both the thumbtray and the film SVG to animate smoothly in sync is difficult, but having them together like this makes it really easy.

Have a style in it for the open state, like this:

```
.panel-sidetray {

      .sidetray-opened & {
             transform: translate(0);
      }
}
```

I will then toggle that style on body (so I can also dim the page etc).